Framework/Algorithms/test/LorentzCorrectionTest.h

```diff
@@ -68,12 +68,9 @@ class LorentzCorrectionTest : public CxxTest::TestSuite {
     auto workspace = Create2DWorkspaceBinned(nSpectra, nBins, startX,
                                              deltaX); // Creates histogram data
 
-    auto &ydata = workspace->dataY(0);
-    auto &edata = workspace->dataE(0);
-    for (size_t i = 0; i < ydata.size(); ++i) {
-      ydata[i] = 1;
-      edata[i] = 1;
-    }
+    auto size = workspace->mutableY(0).size();
+    workspace->mutableY(0).assign(size, 1.0);
+    workspace->mutableE(0).assign(size, 1.0);
```

Comment:
How about adding `operator=(const double)`? You can do this via a new mixin (similar to `Scalable` and `Offsetable`)? This would be useful in many places, I think.